Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM node:alpine
+FROM node:6-alpine
 
-ENV appdir /usr/local/app
+ENV APPDIR /usr/local/app
 
-RUN mkdir -p $appdir
-WORKDIR  $appdir
-COPY . $appdir
+RUN mkdir -p "$APPDIR"
+WORKDIR $APPDIR
+COPY . $APPDIR
 RUN npm install
 
```

Comment:
I think the `MAINTAINER` field has been [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). Maybe

```dockerfile
LABEL maintainer="https://github.com/verdaccio/verdaccio"
```